### PR TITLE
Added Env.getOrFail method.

### PR DESCRIPTION
**Pull request by moltar**
Addresses #847

### CODE REVIEW

File: src/Env/index.js [hunk 1]
```diff
@@ -156,6 +157,31 @@ class Env {
...
+   * @example
+   * ```js
+   * Env.getOrFail('MAIL_PASSWORD')
+   * ```
+   */
+  getOrFail (key) {
+    const val = _.get(process.env, key)
+
+    if (_.isUndefined(val)) {
+      throw GE.RuntimeException.incompleteConfig(`process.env.${key}`, '.env')
```
Comment (thetutlage) [inline src/Env/index.js]: Let's use the `missingEnvKey` exception here as discussed on https://github.com/adonisjs/adonis-framework/issues/847#event-1610545001

**Comment by moltar** [comment]
Pending review, merge and release of the following PR:

https://github.com/adonisjs/adonis-generic-exceptions/pull/3

Then I'll add it as a dependency to this PR and use it.

**Comment by RomainLanz** [comment]
It should be good.

Could you make the requested changes? Thanks!

**Comment by RomainLanz** [comment]
Thanks 😄

**Pull request merged**